tests/pages/catalog/packages/package/PackageTab-cy.js

```diff
@@ -110,6 +110,80 @@ describe("Package Detail Tab", function() {
         expect(result).to.equal(true);
       });
     });
+
+    it("Does not ask for confirmation for certified packages", function() {
+      cy.get(".page-body-content .package-action-buttons button")
+        .contains("Review & Run")
+        .click();
+      cy.get(".modal-body")
+        .contains("Install Community Package")
+        .should("not.exist");
+      cy.get(".modal-full-screen-header-title")
+        .contains("Edit Configuration")
+        .should("exist");
```

Comment:
i'd propose to remove the `.should("exist");`s. they're the defaults expectation and are just noise in most cases.